code/__HELPERS/_macros.dm

```diff
@@ -185,6 +185,9 @@
 	dview_mob.see_invisible = invis_flags; \
 	for(type in view(range, dview_mob))
 
+//get_turf(): Returns the turf that contains the atom.
+//Example: A fork inside a box inside a locker will return the turf the locker is standing on.
+//Yes, this is the fastest known way to do it.
```

Comment:
I would assume that if a faster way is discovered, it would get changed pretty quickly